docs/base/design_sprint.md

```diff
@@ -0,0 +1,51 @@
+# Documento de Design Sprint
+
+## 1. Versionamento
+Versão|Data|Descrição|Autor(es)
+------|----|---------|--------
+1.0   | 02/02/2022 | Criação do documento e adição da introdução e realização de cada etapa | Rafael
+
+## 2. Introdução
+<p style="text-align: justify; text-indent: 20px"> De acordo com [1]: "Design Sprint é um processo de cinco dias para responder a perguntas críticas de negócios por meio de design, prototipagem e teste de ideias com o cliente". Neste sentido, ela é uma boa prática para encurtar o tempo de atividades que demorariam meses em semana(s).
+</p> 
+<p style="text-align: justify; text-indent: 20px">É um processo pensado para ser realizado em cinco dias tendo uma atividade para cada dia [2] e [3]:</p> 
+
```

Comment:
- "De acordo com [1]" deveria ser "De acordo com Tal Pessoa (ANO) [1]"
- "tendo uma atividade para cada dia [2] e[3]", o que seriam esses 2 e 3 ? Deixa explícito quem nem o comentario de cima

Reply:
Ué, agora não entendi o pq deixar explícito, essas referências não são justamente para evitar deixar explícito ?? Pelo menos é o que vejo em diversos artigos científicos, se for pra deixar específico basta colocar uma bibliografia não ?

Reply:
Me expressei mal, a ideia é continuar à utilizar a numeração só que manter o fluxo da leitura sem uma quebra brusca como aconteceu nos pontos que eu destaquei.
Nos pontos, principalmente sobre as referências 2 e 3, quando você está lendo inicialmente o interesse está voltado no conteúdo e não na referência, por isso que utilizar bruscamente o [2] e [3] sem uma indiciação de nome ou conteúdo acaba quebrando a linha de raciocinio de quem está lendo.

Reply:
Perfeito então, concordo contigo, vou arrumar !

Reply:
Fixed.